app/static/js/remove_data.js

```diff
@@ -0,0 +1,75 @@
+const slides = [
```

Comment:
Having a large page that you have to scroll through seemed like it would be overwhelming. I was going for something like those "onboarding flows" where it walks you through each step, asking you to take action (grant permission, sign up, etc) along each step of the way. Except this is more of an "offboarding flow" i guess haha